### PR TITLE
fix: limit total of PATs a user can have

### PR DESCRIPTION
**Pull request by nunogois**
https://linear.app/unleash/issue/2-367/rate-limiting-absent-on-creatingadding-token

### CODE REVIEW

File: src/lib/services/pat-service.ts [hunk 1]
```diff
@@ -67,6 +69,12 @@ export default class PatService {
             throw new BadDataError('The expiry date should be in future.');
         }
 
+        if ((await this.patStore.countByUser(userId)) >= PAT_LIMIT) {
+            throw new OperationDeniedError(
+                `Too many PATs ${PAT_LIMIT} already exist for this user.`,
```
Comment (sighphyre) [inline src/lib/services/pat-service.ts:74]: WDYT?

```suggestion
                `Too many PATs (${PAT_LIMIT}) already exist for this user.`,
```

**Review by sighphyre — APPROVED** [review]
Looks good to me!

File: frontend/src/constants/values.ts [hunk 2]
```diff
@@ -1 +1,2 @@
 export const ENV_LIMIT = 15;
+export const PAT_LIMIT = 10;
```
[the remark this answers is not in the document]
Comment (sighphyre) [reply frontend/src/constants/values.ts:2]: Ah, that's a good point, I've gotten too used to seeing these separate. Now that everything is a single project, do you see a reason to keep them separate? I'm not sure I do anymore but I think this is a big change if we collapse them. I think this is worth asking the whole dev team if anyone sees a reason to not collapse them

[the remark this answers is not in the document]
Comment (nunogois) [reply frontend/src/constants/values.ts]: After discussing this in Slack: https://unleash-internal.slack.com/archives/C046J17G6P7/p1667224108156499
This is what I propose for now: https://github.com/Unleash/unleash/pull/2301/commits/c88e0f1e9dcdaf6d75cc24bfbddf65d26cce7ef0

**Review by sighphyre — APPROVED** [review]
LGTM

**Pull request merged**